docker-compose.yml

```diff
@@ -1214,6 +1213,7 @@ services:
     shm_size: *shm-size
     environment:
       <<: *ccache
+      ARROW_BUILD_TESTS: 'OFF'
```

Comment:
Is this needed?
`ci/docker/linux-apt-r.dockerfile` has `ARROW_BUILD_TESTS=OFF`.